src/Symfony/Component/ErrorHandler/ErrorEnhancer/ErrorEnhancerInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ErrorHandler\ErrorEnhancer;
+
+interface ErrorEnhancerInterface
+{
+    /**
+     * @return \Error|null An \Error instance if the class is able to improve the error, null otherwise
+     */
+    public function handleError(\Error $error): ?\Error;
```

Comment:
could it return `FatalErrorInterface`?
(and should we renamed FatalErrorInterface to EnhancedErrorInterface, or just ErrorInterface btw?)